test/inductor/test_torchinductor.py

```diff
@@ -6955,6 +6956,7 @@ def fn(x: torch.Tensor) -> torch.Tensor:
                     fn_opt(inps)
 
         @skipIfRocm
+        @unittest.skipIf(IS_FBCODE, "fbcode system python does not provide torch")
         def test_indirect_device_assert(self):
```

Comment:
This test uses `sys.executable` which should be the path to the currently running interpreter, rather than the system python. Or is that not always the case?